testdata/p4_16_samples_outputs/issue5331_srcinfo.p4-stderr

```diff
@@ -0,0 +1,3 @@
+issue5331_srcinfo.p4(4): [--Wwarn=overflow] warning: x << 18: shifting value with 16 bits by 18
+        __e(y);
```

Comment:
This isn't the clearest warning, but it is not clear what should in it -- the 18 comes from inlining and constant folding a call, so there are multiple places in the code that are relevant.  At the very least, though, it should identify the point in the code that has the shift (line 8)

Reply:
That is definitely better, so maybe #5333 should be doing less.  The source info on the shift operation itself should not be changing.

Reply:
Maybe #5333 should only copy the source info if it is not already set to something else?